php.ini-production

```diff
@@ -1396,6 +1396,11 @@ session.cookie_domain =
 ; http://php.net/session.cookie-httponly
 session.cookie_httponly =
 
+; Add SameSite attribute to cookie to help mitigate Cross-Site Request Forgery (CSRF/XSRF)
+; Current valid values are "Lax" or "Strict"
+; https://tools.ietf.org/html/draft-west-first-party-cookies-07
+session.cookie_samesite =
```

Comment:
should we be more strict per default with the default cookie settings, to be more secure per default (e.g. with php8)?

Reply:
I think that would be a good idea. At least it deserves an RFC.